pkg(scope): com.google.android.tungsten.setupwraith should not be removed for Android TV (for Sony Bravia atleast)

### Describe the problem

removing this recommended package crashes system sound settings and makes TV speaker's volume control impossible (remote/app/etc)

here is the relevant logcat of the crash

```logcat
2025-06-16 20:03:34.277  2892-2892  AndroidRuntime          com.realtek.advancesetting           E  FATAL EXCEPTION: main
                                                                                                    Process: com.realtek.advancesetting, PID: 2892
                                                                                                    java.lang.RuntimeException: Unable to create service com.realtek.advancesetting.SpeakersChangeService: java.lang.SecurityException: Failed to find provider com.google.android.tungsten.setupwraith for user 0; expected to find a valid ContentProvider for this authority
                                                                                                    	at android.app.ActivityThread.handleCreateService(ActivityThread.java:4204)
                                                                                                    	at android.app.ActivityThread.access$1500(ActivityThread.java:237)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1932)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106)
                                                                                                    	at android.os.Looper.loop(Looper.java:223)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7656)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
                                                                                                    Caused by: java.lang.SecurityException: Failed to find provider com.google.android.tungsten.setupwraith for user 0; expected to find a valid ContentProvider for this authority
                                                                                                    	at android.os.Parcel.createExceptionOrNull(Parcel.java:2374)
                                                                                                    	at android.os.Parcel.createException(Parcel.java:2358)
                                                                                                    	at android.os.Parcel.readException(Parcel.java:2341)
                                                                                                    	at android.os.Parcel.readException(Parcel.java:2283)
                                                                                                    	at android.content.IContentService$Stub$Proxy.registerContentObserver(IContentService.java:1229)
                                                                                                    	at android.content.ContentResolver.registerContentObserver(ContentResolver.java:2627)
                                                                                                    	at android.content.ContentResolver.registerContentObserver(ContentResolver.java:2615)
                                                                                                    	at com.realtek.advancesetting.SpeakersChangeService.onCreate(SpeakersChangeService.java:825)
                                                                                                    	at android.app.ActivityThread.handleCreateService(ActivityThread.java:4192)
                                                                                                    	at android.app.ActivityThread.access$1500(ActivityThread.java:237) 
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1932) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106) 
                                                                                                    	at android.os.Looper.loop(Looper.java:223) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7656) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
                                                                                                    Caused by: android.os.RemoteException: Remote stack trace:
                                                                                                    	at com.android.server.content.ContentService.registerContentObserver(ContentService.java:355)
                                                                                                    	at android.content.IContentService$Stub.onTransact(IContentService.java:482)
                                                                                                    	at android.os.Binder.execTransactInternal(Binder.java:1154)
                                                                                                    	at android.os.Binder.execTransact(Binder.java:1123)
```

Android TV OS: 11
Model: K-55S25 (Bravia VU31)

### You have a solution?

restore the package

### Provide logs

[UAD_20250616.log](https://github.com/user-attachments/files/20759436/UAD_20250616.log)

### Acknowledgements

- [x] This issue is not a duplicate of an existing one.
- [x] I have chosen an appropriate title.
- [x] All requested information has been provided properly.

--- a/resources/assets/uad_lists.json
+++ b/resources/assets/uad_lists.json
@@ -33949,11 +33949,11 @@
   },
   "com.google.android.tungsten.setupwraith": {
     "list": "Oem",
-    "description": "First boot setup TV.\nhttps://play.google.com/store/apps/details?id=com.google.android.tungsten.setupwraith&hl=en&gl=US",
+    "description": "First boot setup TV.\nRemoving it crashes sound related system settings and breaks volume control on Sony Bravia TV.\nhttps://play.google.com/store/apps/details?id=com.google.android.tungsten.setupwraith&hl=en&gl=US",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Recommended"
+    "removal": "Expert"
   },
   "com.google.android.tv.axel": {
     "list": "Oem",

